toolchains/kubectl/README.md

```diff
@@ -91,6 +93,56 @@ source repository is compatible with. Look at the `http_archive` invocation in
 https://github.com/kubernetes/kubernetes/blob/{k8s_commit}/build/root/WORKSPACE
 for the `@io_kubernetes_build` to get the commit pin, sha256 and prefix values.*
 
+### Download a custom kubectl binary
+
+If you want to download a standard binary released by the kubernetes project,
+get the URL and SHA256 for the binary for your platform from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-using-curl).
+For example, if you want to use kubectl v1.10.0 on a x86 64 bit Linux platform,
+add to your `WORKSPACE` file the following lines (Note: The call to
+`kubectl_configure` must be before the  call to `k8s_repositories`):
+
+```python
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "io_bazel_rules_docker",
+    commit = "{HEAD}",
+    remote = "https://github.com/bazelbuild/rules_docker.git",
+)
+
+load(
+  "@io_bazel_rules_docker//container:container.bzl",
+  container_repositories = "repositories",
+)
+
+container_repositories()
+
+# This requires rules_docker to be fully instantiated before
+# it is pulled in.
+git_repository(
+    name = "io_bazel_rules_k8s",
+    commit = "{HEAD}",
+    remote = "https://github.com/bazelbuild/rules_k8s.git",
+)
+
+load("//toolchains/kubectl:kubectl_configure.bzl", "kubectl_configure")
```

Comment:
this load needs to be:
`load("@io_bazel_rules_k8s//toolchains/kubectl:kubectl_configure.bzl", "kubectl_configure")`